cve_bin_tool/cli.py

```diff
@@ -140,6 +141,19 @@ def main(argv=None):
         choices=["low", "medium", "high", "critical"],
         help="minimum CVE severity to report (default: low)",
     )
+    output_group.add_argument(
+        "-a",
+        "--append",
+        nargs="?",
+        const=True,
+        help="save output as intermediate report in json format",
+    )
+    output_group.add_argument(
+        "-t",
+        "--tag",
+        action="store",
+        help="add a tag for differentiate between intermediate reports",
```

Comment:
```suggestion
        help="add a tag to group intermediate reports in a directory",
```

Reply:
I am currently not saving intermediate reports in separate directories on the basis of the tag. This key will just add a tag to the metadata of the report so that the user might be able to differentiate while handling multiple reports.
I think it should be something like  "add a unique tag to differentiate b/w different intermediate reports"

Reply:
Okay, so for now you can then skip this change. But surely the user will find filtering reports on the basis of tag very useful.